tests/ts_cluster_updowngrade_select_aggregation_SUITE.erl

```diff
@@ -0,0 +1,243 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2016 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+-module(ts_cluster_updowngrade_select_aggregation_SUITE).
+
+-export([
+         suite/0,
+         init_per_suite/1,
+         end_per_suite/1,
+         all/0,
+         run_this_test/1
+        ]).
+
+-ifdef(EQC).
```

Comment:
Not convinced this should be a common test - a suite of 1 tests - but if we are strategically going to move to CT it would make sense to have it so...

But I am not convinced that we are going in that direction - its not entirely clear what the point of this comment is, except to get other peoples opinions...